src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -117,6 +117,14 @@ class SaveInProgressIntro extends React.Component {
             savedAt && format(savedAt, "MMMM d, yyyy', at' h:mm aaaa z");
 
           const H = `h${this.props.headingLevel}`;
+          const continueMsg = (
+            <p>
+              You can continue {appAction} now
+              {appContinuing && ` ${appContinuing}`}, or come back later to
+              finish your {appType}.
+            </p>
+          );
```

Comment:
My only suggestion is to use PascalCase when naming a react component. e.g. ContinueMsg, FormReactivationAlert, to improve clarity in intent and consistency. Vanilla javascript declarations can remain in camelCase.